api/clients/disperser_client.go

```diff
@@ -78,6 +78,8 @@ type disperserClient struct {
 	//       instead of a real network connection for eg.
 	conn   *grpc.ClientConn
 	client disperser_rpc.DisperserClient
+
+	accountant Accountant
```

Comment:
should this be removed?

Reply:
I thought it would be okay to leave as is, but removing for clarity